workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -34,6 +34,20 @@ barcodes = ['10Xv2': '737K-august-2016.txt',
 // supported single cell technologies
 tech_list = barcodes.keySet()
 
+// file paths
+index_path = "${params.ref_dir}/${params.index_dir}/${index_names_map[params.index_type]}"
+index_prefix = "${params.ref_dir}/${params.annotation_dir}"
+t2g_path = "${index_prefix}/${t2g_map[params.index_type]}"
+  
+// if using splici and cr-like use the 3 column t2g file for alevin-fry quant
+if(params.resolution == 'cr-like' && params.index_type == 'splici'){
+    t2g_quant_path = "${index_prefix}/${params.t2g_3col}"
+    use_mtx = true
+} else{
+    t2g_quant_path = t2g_path
+    use_mtx = false
+}
```

Comment:
I like this, but now I am inspired to make it a param (this is a bad idea, ignore it): 

But just to say what it would look like, you would have `params.use_mtx = false` at the top, then override it only in first case here (leaving the second case to whatever the user specified).

If I thought there was a chance we would want this workflow to allow mtx output at request, I would say it was a good idea, but we don't so it probably isn't.